src/content/docs/Build Your Project/build-commands.md

```diff
@@ -65,42 +78,56 @@ $ tree .
 ```
 ## build
 
-`build [target]`
+```bash
+build [target]
+```
 
 Build the project in the current path. It doesn't matter where in the project structure you are. 
 
 The built-in templates define two targets: `debug` (which is the default) and `release`.
 
 ## clean
 
-`clean`
+```bash
+clean
```

Comment:
This should not be marked "bash" same with some other stuff above and below.

Reply:
No problem.

I added c3c to the start of the commands so they were more obvious what they were for

Also found 
`c3c docs [<target>]` 
needed to be changed to:
`c3c directives [<target>]`

Reply:
Directives??

Reply:
![image](https://github.com/user-attachments/assets/b316871d-f992-427a-8891-bcdaffcd2e81)